articles/active-directory/manage-apps/grant-admin-consent.md

```diff
@@ -80,10 +80,11 @@ https://login.microsoftonline.com/{organization}/adminconsent?client_id={client-
 where:
 
 - `{client-id}` is the application's client ID (also known as app ID).
-- `{organization}` is the tenant ID or any verified domain name of the tenant you want to consent the application in. You can use the value `common`, which will cause the consent to happen in the home tenant of the user you sign in with.
+- `{organization}` is the tenant ID or any verified domain name of the tenant you want to consent the application in. You can use the value `organizations`, which will cause the consent to happen in the home tenant of the user you sign in with.
 
 As always, carefully review the permissions an application requests before granting consent.
 
+For more information regarding constructing the tenant-wide admin consent URL, see [Grant tenant-wide admin consent to an application](../develop/v2-admin-consent.md)
```

Comment:
```suggestion
For more information on constructing the tenant-wide admin consent URL, see [Admin consent in the Microsoft identity platform](../develop/v2-admin-consent.md)
```

Reply:
I think the wording here needs to reflect what the article you're linking to explains. Feel free to sign-off after considering the suggestion. Thanks